File: assets/c/math-operation.c
// Summation, subtraction, division, multiplication of A and B with and without user input.

#include <stdio.h>

int main() {
  int A, B;
  printf("Enter first number: ");
  scanf("%d", &A);
  printf("Enter second number: ");
  scanf("%d", &B);

  printf("Summation = %d\n", A + B);
  printf("Subtraction = %d\n", A - B);
  printf("Multiplication = %d\n", A * B);
  printf("Divide = %d\n", A / B);

  return 0;
}

File: assets/js/math-operation.js
// Problem: Summation, subtraction, division, multiplication of A and B with and without user input.

let A = +prompt('Enter your first number: '),
    B = +prompt('Enter your second number: '),
    sum = A + B,
    sub = A - B,
    mul = A * B,
    div = A / B;

console.log(`User first input: ${A}`);
console.log(`User first input: ${B}`);
console.log(`Summation = ${sum}`);
console.log(`Subtraction = ${sub}`);
console.log(`Multiplication = ${mul}`);
console.log(`Division = ${div}`);
